assets/js/index.js

```diff
@@ -154,6 +155,17 @@ $(document).ready(function () {
     }
   });
 
+  $("body").click(function(event) {
+    var sidebar = $(".sidebar") ;
+    var targetElement = $(event.target);
+    if( $(window).width() <= 767 && !targetElement.hasClass("sidebar") && !targetElement.hasClass("fas") && sidebar.hasClass("animate__slideInLeft") && (!sidebar.hasClass("hide-sidebar"))){
```

Comment:
Replace `!targetElement.hasClass("fas")` with `!targetElement.hasClass("sidebar") && !targetElement.parents('.sidebar').length`. This ensures that when clicking on the sidebar or any of its child elements, this event listener will not continue.